superset-frontend/src/dashboard/components/SliceHeaderControls/index.tsx

```diff
@@ -309,8 +310,8 @@ class SliceHeaderControls extends React.PureComponent<
 
         {supersetCanShare && (
           <ShareMenuItems
-            copyMenuItemTitle={t('Copy chart URL')}
-            emailMenuItemTitle={t('Share chart by email')}
+            copyMenuItemTitle={t('Copy permalink to clipboard')}
```

Comment:
I'm not sure if using a word "permalink" brings value to the end user. I feel like it might be confusing, and simply saying "link" or "URL" makes the intention clearer. I'd suggest asking a product manager for an opinion on that.

Reply:
The inspiration here is coming from Github and Shortcut.

<img width="1752" alt="Screen Shot 2022-03-08 at 10 02 21 AM" src="https://user-images.githubusercontent.com/70410625/157685290-ae884bf3-f637-4662-b04c-47aaecd5bc37.png">
<img width="671" alt="Screen Shot 2022-03-08 at 10 03 35 AM" src="https://user-images.githubusercontent.com/70410625/157685301-99f5a27d-48b5-4a0f-b32a-5753a1c0aadc.png">

Reply:
Agreed - I know @michael-s-molina did some investigative work and found many products use the term. Here's GitHub:
<img width="239" alt="image" src="https://user-images.githubusercontent.com/33317356/157690243-f402a325-64ed-4bd2-9a53-395186fd263a.png">

One other thing we may want to consider is removing "to clipboard", as it's pretty much self evident.